deliverables/README.md

```diff
@@ -21,48 +21,117 @@ For projects on Code Academy, or other external exercises, please just paste a l
 
 ### Section 1: The Front End
 #### Required - Answer these questions in the deliverables file, and complete the following. 
-What is the role of HTML in a web page?  
-What is the role of CSS in a web page?  
-What is the role of Javascript in a web page?  
+What is the role of HTML in a web page?
+
+HTML is the skeloton and content of a web page.  
+
+What is the role of CSS in a web page?
+
+CSS is the styling and make up of a page
+
+What is the role of Javascript in a web page?
+
+How the page functions. The interaction with the page.  
+
 
 ---
 
 ### Section 2: HTML and CSS 101
 
 #### meta
-Why do we separate HTML and CSS?  
+Why do we separate HTML and CSS?
+You can do most CSS in the HTML (in line CSS), but it is easier to visualize and organize code if they are separated into "mark up" and "make up". Also by using a separate CSS file, you can have more complex and specifc instructions so there is less work. 
+
+
 
 #### HTML
-What are classes and IDs (and how are they different)?  
+What are classes and IDs (and how are they different)?
+Classes are groupings of content that you want the same CSS changes to be applied to. If you create (li class"fancy") this line along with all the other members of that class, will be treated the same. 
```

Comment:
Here you can use backticks (`` ` ``) to designate code.  Ex:  `<li class="fancy">`